www/bust01/Seminární práce/css/main.css

```diff
@@ -0,0 +1,870 @@
+* {
+    box-sizing:border-box;
+    font-family: 'Century Gothic';
+}
+
+header > .nadpis {
+    margin : 0 500px;
+}
+
+.pozadi {
+    color: white;
+    min-height: 600px;
+    padding-top: 60px;
+    background-image: url('../img/pozadi.png');
+    background-size: cover;
+    background-position: center;
+    margin-bottom: 20px;
+}
+
+.logo {
+    background-image: url('../img/czech_team.png');
+    margin-left:190px;
+    margin-bottom: 30px;
+    width: 300px;   
+    height: 150px;
+    background-repeat: no-repeat;
+    background-position: center;
+    float:left;
+}
+
+nav ul li {
+    float: left;
+    padding: 15px 25px;
+    list-style: none;
+    color:white;
+    background-color: #d61149;
+    border-right: 1px solid white;  
+}
+
+nav ul li:hover{
+    background-color:#bad142;
+    font-weight: bold;
+}
+
+nav ul {
+    margin-left: 400px;
+    padding: 0 0;
+    transform: skewX(-20deg);
+}
+
+a:link {
+    text-decoration: none; 
+}
+
+nav ul li a {
+    transform: skewX(20deg);
+    display: inline-block;
+    padding-right: 25px;
+    font-size: 18px;
+}
+
+a {
+    color: white;
+}
+
+.clear {
+    clear:both;
+     float:none;
+     width:0;
+     height:0;
+     padding:0;
+}
+
+.ukazky, .odkazy, .sportovci, .sporty_zoh, .sporty_loh  {
```

Comment:
proc nepouzivas anglicke pojmy?

Reply:
Nevím, lépe si to poté pamatuji. Pokud to vadí, tak to změním.